src/components/Footer.vue

```diff
@@ -2,14 +2,16 @@
   <div class="container-fluid footer p-0 page_404">
     <div class="row footer justify-content-center m-0">
       <div class="col-3 footer">
-        <a href="https://www.cornelldti.org" class="footer"> Cornell DTI @ {{ getYear() }}</a>
+        <a href="https://www.cornelldti.org/projects/courseplan/" class="footer underline">
+          Cornell DTI @ {{ getYear() }}</a
+        >
       </div>
       <div class="col-3">
-        <p class="footer">Built with LOVE</p>
+        <p class="footer">Built with ❤️</p>
       </div>
       <div class="col-3 footer">
         <a
-          class="footer"
+          class="footer underline"
```

Comment:
Looks like adding this class makes the underline on the footer links permanent instead of showing up on hover. Do we not want this to be a hover state, and if not, do we want any other sort of hover state? Tagging @einc on this because I'm not sure what the answer is

Reply:
Design wise, let's add hover states to the links and set them lighter.
Copying from Theresa: "By default, CP implements hovering on buttons (not divs) by setting the opacity to 0.95 on hover. See _global.scss."